screenshots.js

```diff
@@ -95,14 +109,23 @@ const TMP_CONFIG = { postfix: '.png' };
 			log('-- no diff');
 		}
 	}
+	function onResponse(page, isMaster) {
+		const pageUrls = Object.keys(config).map(path => `http://${PR}.talend.surge.sh/${path}`);
+		page.on('response', async res => {
+			if (pageUrls.includes(res.url()) && res.status() === 404) {
```

Comment:
Why do you check if the requested url is one of the PR's surge path ?

Reply:
I remember this is because the response is not only the main page response, but also from assets etc. And it's only interesting to know if the main request is 404.